internal/naming/naming.go

```diff
@@ -93,10 +93,10 @@ func BuildDeploymentName(componentType consts.ComponentType) string {
 	}.String()
 }
 
-func BuildDaemonSetName(componentType consts.ComponentType) string {
+func BuildDaemonSetName(componentName string) string {
```

Comment:
This is a specific for NodeConfigurator DaemonSetName or not? I'm asking because name of a method is common but u used here consts.NodeConfiguratorName